boards/.gitignore

```diff
@@ -0,0 +1 @@
+/__pycache__/
```

Comment:
In my build system I found that during build, my Python environment was writing python cache information.  I can't imagine that we would want that in our build system so added a gitignore for it.  If that is wrong, we can remove it.